packages/NodeTypeResolver/Node/AttributeKey.php

```diff
@@ -23,6 +23,8 @@ final class AttributeKey
     public const VIRTUAL_NODE = 'virtual_node';
 
     /**
+     * @deprecated Use AbstractScopeAwareRector->refactorWithScope() instead
```

Comment:
I prefer to not deprecate, use `@internal` is more make sense, as needed if it used inside traverser, like this one:

https://github.com/rectorphp/rector-downgrade-php/blob/ba00f59eafc80698ce25e00678b9cd47cb8b7594/rules/DowngradePhp72/Rector/FuncCall/DowngradeStreamIsattyRector.php#L115-L117